src/toil/cwl/cwltoil.py

```diff
@@ -323,6 +387,16 @@ def makeJob(tool, jobobj, **kwargs):
         wfjob.addFollowOn(followOn)
         return (wfjob, followOn)
     else:
+        # get_requirement
+        resourceReq, _ = tool.get_requirement("ResourceRequirement")
+        for req in ("coresMin", "coresMax", "ramMin", "ramMax",
+                     "tmpdirMin", "tmpdirMax", "outdirMin", "outdirMax"):
+            r = resourceReq.get(req)
+            if isinstance(r, string_types) and "$(" in r or "${" in r:
```

Comment:
From the logs (I forget if you have permission to see them, my apologies for the spam if you do):
```
Test failed: /mnt/ephemeral/workspace/toil-pull-requests/venv/bin/cwltoil --outdir=/mnt/ephemeral/tmp/tmpFSMzsM --quiet v1.0/bwa-mem-tool.cwl v1.0/bwa-mem-job.json
General test of command line generation
Returned non-zero
Traceback (most recent call last):
  File "/mnt/ephemeral/workspace/toil-pull-requests/venv/bin/cwltoil", line 9, in <module>
    load_entry_point('toil', 'console_scripts', 'cwltoil')()
  File "/mnt/ephemeral/workspace/toil-pull-requests/src/toil/cwl/cwltoil.py", line 810, in main
    tmpdir=os.path.realpath(outdir))
  File "/mnt/ephemeral/workspace/toil-pull-requests/src/toil/cwl/cwltoil.py", line 395, in makeJob
    if isinstance(r, string_types) and "$(" in r or "${" in r:
TypeError: argument of type 'int' is not iterable
```
Which is odd and doesn't make sense to me :-/